src/resonanceReconstruction/rmatrix/legacy.hpp

```diff
@@ -0,0 +1,5 @@
+namespace legacy {
```

Comment:
I'm not sure what is legacy about this? Please explain.

Reply:
As in legacy resonance reconstruction that is going to be phase out. SLBW and MLBW will go in there as well. The treatment in all these cases is specifically ENDF and offers little to no generic processing like the LRF7.